src/Symfony/Component/HttpClient/HttpClient.php

```diff
@@ -23,17 +24,20 @@
 final class HttpClient
 {
     /**
-     * @param array $defaultOptions     Default requests' options
-     * @param int   $maxHostConnections The maximum number of connections to a single host
+     * @param LoggerInterface $logger
+     * @param array           $defaultOptions     Default requests' options
+     * @param int             $maxHostConnections The maximum number of connections to a single host
+     *
+     * @return HttpClientInterface
      *
      * @see HttpClientInterface::OPTIONS_DEFAULTS for available options
      */
-    public static function create(array $defaultOptions = [], int $maxHostConnections = 6): HttpClientInterface
+    public static function create(LoggerInterface $logger, array $defaultOptions = [], int $maxHostConnections = 6): HttpClientInterface
```

Comment:
We shouldn't make it a mandatory argument - providing no logger should be allowed to me (same in all client implementations)